Fix for #1506 (Touchscreen calib) and 3db reduction on 512FFT

db4ple opened the pull request:
two unrelated fixes:
- fix dbm display for large fft by reducing value by 3db so it
  is now identical to 256fft value
- fix regression in calibration (no manual touch possible, broken calibration)

sp9bsl commented on mchf-eclipse/drivers/ui/ui_driver.c:
```diff
@@ -5499,51 +5505,47 @@ static bool UiDriver_TouchscreenCalibration()
 #ifdef Touch_ShowTestscreen
-	    UiLcdHy28_PrintTextCentered(2, 195, MAX_X-4, "Press BAND+ and BAND-\n"
+	    UiLcdHy28_PrintTextCentered(2, 195, MAX_X-4, "Press BAND+and Band-\n"
 	            "to run drawing on screen\n"
```
Hi Danilo,
typo here? BAND- xchanged with Band-?

db4ple commented:
Sorry, that was just a unintended side effect, will fix that.

db4ple committed:
Fix for #1506 (Touchscreen calib) and 3db reduction on 512FFT
two unrelated fixes:
- fix dbm display for large fft by reducing value by 3db so it
  is now identical to 256fft value
- fix regression in calibration (no manual touch possible, broken calibration)

db4ple commented:
Fixed small typo
